index.d.ts

```diff
@@ -1,25 +1,6 @@
-// Referencing the default lib web api typings
-/// <reference lib="dom" />
+/* tslint:disable */
+/* eslint-disable */
 
-declare module "node-web-audio-api" {
-    export import AudioContext = globalThis.AudioContext;
-    export import OfflineAudioContext = globalThis.OfflineAudioContext;
-    export import AudioBuffer = globalThis.AudioBuffer;
-    export import PeriodicWave = globalThis.PeriodicWave;
-    export import MediaStreamAudioSourceNode = globalThis.MediaStreamAudioSourceNode;
-    export import AnalyserNode = globalThis.AnalyserNode;
-    export import AudioBufferSourceNode = globalThis.AudioBufferSourceNode;
-    export import BiquadFilterNode = globalThis.BiquadFilterNode;
-    export import ChannelMergerNode = globalThis.ChannelMergerNode;
-    export import ChannelSplitterNode = globalThis.ChannelSplitterNode;
-    export import ConstantSourceNode = globalThis.ConstantSourceNode;
-    export import ConvolverNode = globalThis.ConvolverNode;
-    export import DelayNode = globalThis.DelayNode;
-    export import DynamicsCompressorNode = globalThis.DynamicsCompressorNode;
-    export import GainNode = globalThis.GainNode;
-    export import IIRFilterNode = globalThis.IIRFilterNode;
-    export import OscillatorNode = globalThis.OscillatorNode;
-    export import PannerNode = globalThis.PannerNode;
-    export import StereoPannerNode = globalThis.StereoPannerNode;
-    export import WaveShaperNode = globalThis.WaveShaperNode;
-}
+/* auto-generated by NAPI-RS */
+
+export function runAudioWorklet(): void
```

Comment:
not sure what happened to this file and the other index.js

Reply:
I think it is due to the fact the you used the `#[napi]` macro which is really high level.

I will have a look to align this with the rest of the codebase logic, should not be complicated

Reply:
Ok really strange, the changes are seems to be good but there is some sort of cache that makes these 2 files reappear (or erased) by napi constantly. But if I clone the repo again, everything is good...

Reply:
Just spend 1 hour and half fiddling on that but cannot figure out where this comes from, so I just tried to go brute force and restart from a fresh clone... but the cache seems to be linked on the directory name, that's GRRRR!

If I just clone with another directory name, everything is fine...

Reply:
Ok quite insane, step I made to have it work properly (seems the reboot was the big deal, because other steps alone didn't work but not really sure neither):

```
rm -Rf target/
rm -Rf node_modules/
cargo cache -a # using https://crates.io/crates/cargo-cache
rm -Rf ~/.npm/_cacache/

# reboot computer now!

npm install 
npm run build
```

And everything behaved as expected finally...